src/_tests/libs/runtime-utils.test.js

```diff
@@ -0,0 +1,189 @@
+import { tools } from 'src/components/notebook-utils'
+import { getCurrentApp, getCurrentAppIncludingDeleting, getCurrentPersistentDisk, getDiskAppType } from 'src/libs/runtime-utils'
+
+
+const cromwellRunning = {
+  appName: 'terra-app-83f46705-524c-4fc8-xcyc-97fdvcfby14f',
+  appType: 'CROMWELL',
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-28T20:28:01.998494Z', destroyedDate: null, dateAccessed: '2021-11-28T20:28:01.998494Z'
+  },
+  diskName: 'saturn-pd-693a9707-634d-4134-bb3a-xyz73cd5a8ce',
+  errors: [],
+  googleProject: 'terra-test-e4000484',
+  kubernetesRuntimeConfig: { numNodes: 1, machineType: 'n1-highmem-8', autoscalingEnabled: false },
+  labels: {},
+  proxyUrls: { 'cromwell-service': 'https://leonardo-fiab.dsde-dev.broadinstitute.org/fd0cfbb14f/cromwell-service/swagger/cromwell.yaml' },
+  status: 'RUNNING'
+}
+
+// Newer than cromwellRunning.
+const cromwellProvisioning = {
+  appName: 'terra-app-73f46705-524c-4fc8-ac8c-07fd0cfbb14f',
+  appType: 'CROMWELL',
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-29T20:28:01.998494Z', destroyedDate: null, dateAccessed: '2021-11-29T20:28:01.998494Z'
+  },
+  diskName: 'saturn-pd-693a9707-634d-4134-bb3a-cbb73cd5a8ce',
+  errors: [],
+  googleProject: 'terra-test-e4000484',
+  kubernetesRuntimeConfig: { numNodes: 1, machineType: 'n1-highmem-8', autoscalingEnabled: false },
+  labels: {},
+  proxyUrls: { 'cromwell-service': 'https://leonardo-fiab.dsde-dev.broadinstitute.org/fd0cfbb14f/cromwell-service/swagger/cromwell.yaml' },
+  status: 'PROVISIONING'
+}
+
+const galaxyRunning = {
+  appName: 'terra-app-69200c2f-89c3-47db-874c-b770d8de737f',
+  appType: 'GALAXY',
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-29T20:19:13.162484Z', destroyedDate: null, dateAccessed: '2021-11-29T20:19:13.162484Z'
+  },
+  diskName: 'saturn-pd-026594ac-d829-423d-a8df-76fe96f5b4e7',
+  errors: [],
+  googleProject: 'terra-test-e4000484',
+  kubernetesRuntimeConfig: { numNodes: 1, machineType: 'n1-highmem-8', autoscalingEnabled: false },
+  labels: {},
+  proxyUrls: { galaxy: 'https://leonardo-fiab.dsde-dev.broadinstitute.org/a-app-69200c2f-89c3-47db-874c-b770d8de737f/galaxy' },
+  status: 'RUNNING'
+}
+
+// Newer than galaxyRunning
```

Comment:
Did you mean
```suggestion
// Older than galaxyRunning
```
?

Why is this comment (and the similar one above) necessary?

Reply:
Ah I think I now understand what you mean by `older/newer` (`createdDate` timestamps). I initially thought you were referring to the lifecycle statuses of compute. Since the comment threw me off, perhaps we could explain a bit more in the comment what we mean by `older/newer` and why it's relevant.

Reply:
Yes, the comments were in reference to the start dates. I was attempting to make some of the test assertions easier to understand so that one didn't have to think through the dates. I'm not sure what else I would say though, as the tests call out the dependency on date newest).